code.json

```diff
@@ -0,0 +1,47 @@
+[
+  {
+    "name": "WMA Docker Java",
+    "organization": "U.S. Geological Survey",
+    "description": "WMA Docker Java - docker image to be extended by USGS WMA Java Docker images requiring a Java Run-time Environment (JRE)",
+    "version": "11.5",
```

Comment:
Remember to change this in the Docker Hub build as well. I can show you where it is if you would like.